add FAQ information provided by Finance (#14664)
netdata/netdata: docs/cloud/manage/view-plan-billing.md

#### 4. Can I get a refund? How?

Payments for Netdata subscriptions are refundable **only** if you cancel your subscription within 14 days of purchase. The refund will be credited to the Credit/Debit Card used for making the purchase. To request a refund, please email us at [billing@netdata.cloud](mailto:billing@netdata.cloud).
            
#### 5. How do I cancel my paid Plan?

Your annual or monthly Netdata Subscription plan will automatically renew until you cancel it. You can cancel your paid plan at any time by clicking ‘Cancel Plan’ from the **Plan & Billing** section under settings. You can also cancel your paid Plan by clicking the _Select_ button under **Community** plan in the **Plan & Billing** Section under Settings. 

#### 6. How can I access my Invoices/Receipts after I paid for a Plan?

You can visit the _Billing Options & Invoices_ in the **Plan & Billing** section under settings in your Netdata Space where you can find all your Invoicing history.

#### 7. Why do I see two separate Invoices? 

> Every time you purchase or renew a Plan, either a Monthly or an Annual, then two  separate Invoices are generated:

Reviewer: ```suggestion
Every time you purchase or renew a Plan, two separate Invoices are generated:
```

>      a) one includes the recurring fees of the Plan you have chosen 

Reviewer: ```suggestion
     - One Invoice includes the recurring fees of the Plan you have chosen 
```

>      - One Invoice includes the recurring fees of the Plan you have chosen 

Reviewer: ```suggestion
- One Invoice includes the recurring fees of the Plan you have chosen 
```

>      - The other Invoice includes your monthly “On Demand - Usage”.

Reviewer: ```suggestion
- The other Invoice includes your monthly “On Demand - Usage”.
```

>      b) the other one includes your monthly “On Demand - Usage”.

Reviewer: ```suggestion
     - The other Invoice includes your monthly “On Demand - Usage”.
```

For the next 24 hours, you will be able to use all your current notification method configurations. After 24 hours, any of the notification method configurations that aren't available on your space's plan will be automatically disabled.

Cancellation might affect users in your Space. Please check what roles are available on the [Community plan](https://learn.netdata.cloud/docs/nightly/concepts/netdata-plans#areas-impacted-by-plans). Users with unavailable roles on the Community plan will immediately have restricted access to the Space.

 #### 3. Which currencies do you support?

We currently accept payments only in US Dollars (USD). We currently have plans to also accept payments in Euros (EUR), but do not currently have an estimate for when such support will be available.

#### 4. Can I get a refund? How?

Payments for Netdata subscriptions are refundable **only** if you cancel your subscription within 14 days of purchase. The refund will be credited to the Credit/Debit Card used for making the purchase. To request a refund, please email us at [billing@netdata.cloud](mailto:billing@netdata.cloud).
            
#### 5. How do I cancel my paid Plan?

> Your annual or monthly Netdata Subscription plan will automatically renew until you cancel them. You can cancel your paid plan at any time by clicking ‘Cancel Plan’ from the **Plan & Billing** section under settings. You can also cancel your paid Plan by clicking the _Select_ button under **Community** plan in the **Plan & Billing** Section under Settings. 

Reviewer: ```suggestion
Your annual or monthly Netdata Subscription plan will automatically renew until you cancel it. You can cancel your paid plan at any time by clicking ‘Cancel Plan’ from the **Plan & Billing** section under settings. You can also cancel your paid Plan by clicking the _Select_ button under **Community** plan in the **Plan & Billing** Section under Settings. 
```

> 

Reviewer: ```suggestion
```
